Sources/TuistSigning/Certificate/CertificateParser.swift

```diff
@@ -77,18 +77,36 @@ final class CertificateParser: CertificateParsing {
         return Certificate(
             publicKey: publicKey,
             privateKey: privateKey,
+            fingerprint: fingerprint,
             developmentTeam: developmentTeam,
             name: name.sanitizeEncoding(),
-            targetName: targetName,
-            configurationName: configurationName,
             isRevoked: isRevoked
         )
     }
 
+    func parseFingerPrint(developerCertificate: Data) throws -> String {
+        let temporaryFile = try FileHandler.shared.temporaryDirectory().appending(component: "developerCertificate.cer")
+        try developerCertificate.write(to: temporaryFile.asURL)
+
+        return try fingerprint(at: temporaryFile)
+    }
+
     // MARK: - Helpers
 
     private func subject(at path: AbsolutePath) throws -> String {
-        try System.shared.capture("openssl", "x509", "-inform", "der", "-in", path.pathString, "-noout", "-subject")
+        do {
+            return try System.shared.capture("openssl", "x509", "-inform", "der", "-in", path.pathString, "-noout", "-subject")
+        } catch {
+            throw CertificateParserError.fileParsingFailed(path)
```

Comment:
maybe we could include the error message from the `openssl` command? Otherwise, it won't be really obvious to the user *why* the parsing failed